### PR TITLE
runltplite.sh: Adding SKIPFILE option

### PR DESCRIPTION
**Pull request by sathnaga**
Though it already runs subtest of tests with a
defined scenario file, it would be good to have
the skip tests options for the user for more flexibility
at run time.

### CODE REVIEW

**Comment by sathnaga** [comment]
@metan-ucw can you help to get this patch pushed, or point me to right person otherwise.
Thanks in advance.

**Comment by yaneurabeya** [comment]
Please add "Signed-off-by: <...>" to your commit message. Thanks.

**Comment by yaneurabeya** [comment]
Note: this can be easily done by running `git commit --amend -s`

**Comment by sathnaga** [comment]
@yaneurabeya Thanks, Done! :-)

File: runltplite.sh [hunk 1]
```diff
@@ -121,7 +122,7 @@ main()
 
     local scenfile=""
 
-    while getopts c:d:hi:l:m:No:pqr:b:B: arg
+    while getopts c:d:hi:l:m:No:pqrS:b:B: arg
```
Comment (gaowanlong) [inline runltplite.sh]: Any way, you are wrongly removing the option argument of "r"...

Comment (gaowanlong) [inline runltplite.sh]: @sathnaga fixing this bug by PR #80

**Pull request merged**